docs/tutorials/services/color-detection-scuttle.md

```diff
@@ -317,16 +322,19 @@ You could also write some code with a Viam SDK to [make your rover move in a squ
 
 {{< snippet "social.md" >}}
 
-## Full Code
+## Full code
 
 ```python {class="line-numbers linkable-line-numbers"}
 import asyncio
+import random
+import vlc
```

Comment:
I noticed this was missing and we directed people to import them, but I'm a bit confused as to where they're actually being used?  @npentrel